test/commonjs_test.spec.ts

```diff
@@ -1,4 +1,5 @@
-import deliveryPersonPb = require('rules_typescript_proto/test/proto/common/delivery_person_pb');
+import {Timestamp} from 'google-protobuf/google/protobuf/timestamp_pb';
```

Comment:
This proto should be at `<workspace-name>/path/to/proto`, so it should be: `com_google_protobuf/google/protobuf/timestamp_pb`

Reply:
I don't think that's the case here. This syntax would load from the npm module `google-protobuf` rather than the user's own workspace

Reply:
I was going to say that the google-protobuf version has special code for timestamp conversion and stuff, but then went and researched and the protoc compiler itself adds that in here https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/compiler/js/well_known_types_embed.cc

Dig-Doug: do you still want this changed to use the non-npm locally generated proto code here?

Reply:
My impression here was that you're trying to test if you can generate a TS protobuf for a proto defined in an external repository.

E.g., let's say you wanted to use one of the protos defined in the `rules_typescript_proto` repo in your own project. For that case, I'd expect to reference the proto as `rules_typescript_proto/path/to/proto`. So for the code above, you could use protos in the protobuf repo to simulate the case I described above. 

Alternatively, you could also handle `google-protobuf` as a special case and do something different. However, I don't see a reason not to use the locally generated versions. I don't think the WKTs are included in the main `google-protobuf.js` bundle, so there won't be any code duplication. I also think you would need to do some additional work to serve those other WKT protos from the npm library in `ts_devserver`/bundling that you would not need with files generated by bazel.

For those reasons, I'm leaning towards using the locally generated files. I don't have a strong opinion here, so if there's a reason we should be using the bundled versions instead let me know.